build.gradle

```diff
@@ -2,6 +2,7 @@ buildscript {
     repositories {
         mavenCentral()
         jcenter()
+        eventuateMavenRepoUrl.split(',').each { repoUrl -> maven { url repoUrl } }
```

Comment:
How did this build on your machine?

Reply:
You mean before? I was thinking about it, but did not find the answer. I tested it before, I am sure. After pulling from master, it failed. This change fixed the fail.